rocksdb_replicator/replicated_db.cpp

```diff
@@ -372,6 +371,7 @@ void RocksDBReplicator::ReplicatedDB::handleReplicateRequest(
                ++i, iter->Next()) {
             auto result = iter->GetBatch();
             Update update;
+            update.set_seq_no(result.sequence);
```

Comment:
do you need add seqno for all replication request, or, you will only need when leader->rocksobserver

Reply:
We only need it for leader->rocksobserver, but on the leader we do not distinguish between different types of follower... we could add a field in the request type to indicate that we need the sequence number, but to keep it simple, I propose always returning it. What do you think?